.gitignore

```diff
@@ -1 +1,2 @@
 __pycache__
+test/.cache
```

Comment:
Did you remove the folder from git too ?
You could have put just `.cache` this way it ignores `.cache` in all folders.